code/datums/flock/flock.dm

```diff
@@ -755,7 +755,10 @@
 		LAGCHECK(LAG_LOW)
 		for(var/turf/tile in turfs)
 			if(istype(tile, /turf/simulated) && !isfeathertile(tile))
-				flock_convert_turf(tile)
+				if (F)
+					F.claimTurf(flock_convert_turf(tile))
+				else
+					flock_convert_turf(tile)
```

Comment:
```suggestion
				flock_convert_turf(tile)
				if (F)
					F.claimTurf(tile)
```

This might just be pedantic, but. Feel free to just resolve.

Reply:
actually not sure that'd work, since I think `flock_convert_turf` deletes the old turf and returns the new one

Reply:
actually I don't think that'd work, since `flock_convert_turf` deletes the old turf and returns the new one

Reply:
It does, if you search in the code for other replacements similar, there's cases of this. Idk why it works.

Reply:
turfs are wack